### PR TITLE
 XWIKI-6732: Introduce DocumentRenamingEvent and DocumentRenamedEvent events

### PR DESCRIPTION
**Pull request by surli**
* Unregister new listener components in some module tests to avoid
missing dependencies.

### CODE REVIEW

**Comment by surli** [comment]
@tmortagne another batch of tests fixing

File: xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-handlers/xwiki-platform-extension-handler-xar/src/test/java/org/xwiki/extension/xar/internal/handler/WikiCopiedEventListenerTest.java [hunk 1]
```diff
@@ -86,6 +87,15 @@ public void setUp() throws Exception
 
         this.localExtensionDependency1 = this.localExtensionRepository.storeExtension(extensionDependency);
         this.localExtension1 = this.localExtensionRepository.storeExtension(extension);
+
+        this.repositoryUtil.getComponentManager().unregisterComponent(EventListener.class,
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-handlers/xwiki-platform-extension-handler-xar/src/test/java/org/xwiki/extension/xar/internal/handler/WikiCopiedEventListenerTest.java]: I feel that registering a mock of ModelBridge (unless one already exist) in MockitoOldcore like we do for other components would be better and easier for new tests than copy/pasting this everywhere.

Comment (surli) [reply xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-handlers/xwiki-platform-extension-handler-xar/src/test/java/org/xwiki/extension/xar/internal/handler/WikiCopiedEventListenerTest.java]: I agree that it'll be more elegant, I'm working on that. However it won't be enough: here for example, we're not using MockitoOldCore. Now I noticed that I have less mock component to register, than listener to unregister, so I'll go toward this solution

Comment (tmortagne) [reply xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-handlers/xwiki-platform-extension-handler-xar/src/test/java/org/xwiki/extension/xar/internal/handler/WikiCopiedEventListenerTest.java]: > we're not using MockitoOldCore

Sure but in many of the affected test we do and it's even possible some of those not using MockitoOldCore should.

Comment (surli) [reply xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-handlers/xwiki-platform-extension-handler-xar/src/test/java/org/xwiki/extension/xar/internal/handler/WikiCopiedEventListenerTest.java]: > t's even possible some of those not using MockitoOldCore should (or should not use `@AllComponent`).

yep, but I don't necessarily plan to refactor them ;)

**Comment by surli** [comment]
Closed in favor of https://github.com/xwiki/xwiki-platform/pull/1059

**Pull request closed without merging**